lib/iex/test/iex/helpers_test.exs

```diff
@@ -676,6 +679,25 @@ defmodule IEx.HelpersTest do
         assert capture_io(fn -> b(MultipleClauseCallback) end) =~ """
                @callback test(:foo) :: integer()
                @callback test(:bar) :: [integer()]
+
+               @callback test2(:foo) :: integer()
+               """
+
+        IO.inspect IEx.Introspection.get_docs(MultipleClauseCallback, [:callback])
```

Comment:
This prints:

```elixir
[
  {{:callback, :test, 1}, 4, [], :none, %{}},
  {{:callback, :test, 1}, 4, [], :none, %{}},
  {{:callback, :test2, 1}, 6, [], %{"en" => "Another callback"}, %{}}
]
```

I'd expect to see "A callback" docs attached, but not sure how exactly that would look like, not sure if we should duplicate it for all clauses? Or it should be represented in a different way altogether?